packages/drivers/odsp-driver-definitions/README.md

```diff
@@ -3,4 +3,4 @@
 Definitions for @fluidframework/odsp-driver package.
 Currently it contains all the contracts for driver factory.
 
-Though the host is responsible for implementing the IPersistedCache, snapshot cached entries will be disregarded if they are older than 2 days, which is based on `defaultCacheExpiryTimeoutMs`. The ODSP driver will delete all entries if it interacts with such a file.
\ No newline at end of file
+Though the host is responsible for implementing the IPersistedCache, snapshot cached entries will be disregarded if they are older than 2 days, per the maximumCacheDurationMs policy on IDocumentStorageServicePolicies
```

Comment:
Maybe not related to this PR but this point seems to come out of nowhere. There should be some details about IPersistedCache before it talks about how maximumCacheDurationMs affects it.